docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/cleavr.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cleavr Deployment - Strapi Developer Documentation
+description: Learn in this guide how to deploy your Strapi application to popular VPS providers using Cleavr.
+---
+
+# Cleavr
+
+This guide explains how to deploy your Strapi projects to a VPS (virtual private server) using [Cleavr](https://cleavr.io).
+
+Cleavr is a server management console that integrates with multiple VPS providers (Hetzner, Linode, Vultr, DigitalOcean, UpCloud, AWS, and custom) 
+and helps you configure servers to host and deploy your Strapi projects.
+
+Cleavr includes the following features:
+
+- Provision and configure servers ready to host Strapi projects
+- Secure servers and provides free SSL certs
+- Deploy code from GitHub, GitLab, and Bitbucket repositories
+- Auto-installs and configures PM2 (with cluster mode enabled)
+- GitHub Actions integration to build app with no additional configuration required
+
+::: tip
+For more information, consult [Cleavr's Strapi deployment guide](https://docs.cleavr.io/guides/strapi-cms)
+:::
+
+### Prerequisites
```

Comment:
```suggestion
::: prerequisites
```
We have a dedicated, custom callout VuePress component for prerequisites. It should be used to emphasize this section (see our [styleguide](https://handbook.strapi.io/user-success-manual/strapi-documentation-style-guide)).